src/components/molecules/PlayaTime/PlayaTime.tsx

```diff
@@ -21,15 +22,10 @@ dayjs.updateLocale("en", {
 const PlayaTime: React.FC = () => {
   const [currentTime, setCurrentTime] = useState("");
 
-  useEffect(() => {
-    const updateTime = () => {
-      const pt = dayjs().tz("Australia/Sydney");
-      setCurrentTime(pt.format("h:mm a"));
-    };
-    updateTime();
-    const id = setInterval(updateTime, 1000);
-    return () => clearInterval(id);
-  }, []);
+  useInterval(() => {
+    const pt = dayjs().tz("Australia/Sydney");
```

Comment:
OOS/future: we should move this to settings or make it venue config, some time soon.

Reply:
User config IMO

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/14